tests/resolve/pass_ops.out

```diff
@@ -1,5 +1,14 @@
-let |>#0 x#1 f#2 = f#2 x#1
-let <|#3 f#4 x#5 = f#4 x#5
-let add#6 a#7 b#8 = (a#7 + b#8)
-let main#9 =
-  (((2 |>#0 ((+ 3)))) add#6 3)
+pass_ops.ml[4:19 ..4:19]: error
```

Comment:
Can we fix these tests? A lot of the resolve ones are failing, when they really shouldn't be.

Reply:
done